src/components/button/secondaryButton.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by jonlazarini on 14/02/17.
+ */
+import React from 'react';
+// import Button from './index.jsx';
+import PrimaryButton from './primaryButton.jsx';
+import styled from 'styled-components';
+
+
+// Style wrapper for default buttons - Composition
+const SecondaryWrapper = (Comp) => styled(Comp)`
+  margin-top: 0.2em;
+  padding: 0.6em 1.6em;
+  border: 2px solid black;
+  font-size: 0.8em;
+  font-weight: 600;
+  text-transform: lowercase;
+  background: black;
+  color: white;
+  &:hover {
+    background: white;
+    color: black;
+    cursor: pointer;
+    }
+  `;
+
+const SecondaryButton = SecondaryWrapper(PrimaryButton);
```

Comment:
Does this not now create dependency on the PrimaryButton for anything that happens here? Changing a style in the PrimaryButton can break this one.